tests/testthat/test-apply_theme.R

```diff
@@ -1,12 +1,12 @@
-#' @title Specifications add_theme
+#' @title Specifications apply_theme
 #' @section Last updated by:
 #' Tim Treis
 #' @section Last update date:
 #' 07-MAY-2021
 
 # Specifications ---------------------------------------------------------------
 
-#' T1. The `define_theme` function returns an object that has a valid shape for usage in the `add_theme` function.
+#' T1. The `define_theme` function returns an object that has a valid shape for usage in the `apply_theme` function.
```

Comment:
What is a valid shape? Can you be more specific?

Reply:
Will improve the wording :+1: